### PR TITLE
create dump rules command

### PR DESCRIPTION
**Pull request by artjomka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,7 +245,39 @@ EXAMPLES
...
+
+USAGE
+  $ sfdx veloce:dumprules -i <string> -o <string> -m <string> [--json] [--loglevel 
+  trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]
+
+OPTIONS
+  -i, --inputdir=inputdir                                                           (required) Input directory to read
+                                                                                    rule files
+
+  -m, --pricegroupconfiguration=pricegroupconfiguration                             (required) Price group configuration
```
Comment (veloceapps) [inline README.md:265]: pgmap ?

File: README.md [hunk 2]
```diff
@@ -244,7 +245,39 @@ EXAMPLES
...
+  --json                                                                            format output as json
+
+  --loglevel=(trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL)  [default: warn] logging level for
+                                                                                    this command invocation
+
+EXAMPLE
+  $ sfdx veloce:dumprules -i rules -o VELOCPQ__PriceRule__c.csv -m pricegroupconfiguration.txt
+```
+
+_See code: [lib/commands/veloce/dumprules.js](https://github.com/veloceapps/veloce-sfdx/blob/v1.0.42/lib/commands/veloce/dumprules.js)_
```
Comment (veloceapps) [inline README.md:280]: why dumprules and not createrules? same patter as for createconfiguration?

File: src/commands/veloce/dumprules.ts [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+const fs = require('fs');
+const createCsvWriter = require('csv-writer').createObjectCsvWriter;
+/* tslint:enable */
+
+// Initialize Messages with the current plugin directory
+Messages.importMessagesDirectory(__dirname);
+
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is using the messages framework can also be loaded this way.
+const messages = Messages.loadMessages('veloce-sfdx', 'dumprules');
```
Comment (veloceapps) [inline src/commands/veloce/dumprules.ts:14]: createrules action? to be consistent?

File: src/commands/veloce/dumprules.ts [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+export default class Org extends SfdxCommand {
+  public static description = messages.getMessage('commandDescription');
+
+  public static examples = [
+    '$ sfdx veloce:dumprules -i rules -o VELOCPQ__PriceRule__c.csv -m pricegroupconfiguration.txt'
```
Comment (veloceapps) [inline src/commands/veloce/dumprules.ts:20]: pgmap.json ?
why txt?

File: README.md (hunk 2 above)
Comment (veloceapps) [inline README.md:280]: add pgmap.json example? to output?

Comment (artjomka) [reply README.md]: > why dumprules and not createrules? same patter as for createconfiguration?

renamed to `packconfig` and `packrules` commands

File: src/commands/veloce/dumprules.ts (hunk 4 above)
Comment (artjomka) [reply src/commands/veloce/dumprules.ts]: > pgmap.json ?
> why txt?

changed to `pgmap.json`

**Pull request merged**